package.json

```diff
@@ -9,7 +9,15 @@
     "jquery": "^3.5.1"
   },
   "devDependencies": {
-    "cypress": "^13.6.6"
+    "cypress": "^13.6.6",
+    "eslint": "^8.57.0",
+    "stylelint": "^16.2.1",
+    "stylelint-config-standard": "^36.0.0"
+  },
+  "scripts": {
+    "lint:js": "eslint js/*.js",
+    "lint:css": "stylelint '**/*.css' --fix",
```

Comment:
![image](https://github.com/andrewsemchism/flexboxfroggy/assets/26263425/b50ddc31-8be8-4de3-a598-6b18e592c594)
```suggestion
    "lint:css": "stylelint **/*.css --fix",
```

I'm on Windows using bash terminal (I think that might make a difference) and it only works when I remove these quotes.

Reply:
Oh yeah possible, for me it's working without the quotes as well. So I can move them!

Reply:
Perfect thanks.